src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/confirm/containers/Confirm.tsx

```diff
@@ -2,9 +2,8 @@
 import * as React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { RouteChildrenProps, withRouter } from 'react-router';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { createMuiTheme } from '@material-ui/core';
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
<3

Reply:
I wonder if more of these could be removed by adding what's mentioned in the bottom of this .md?
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md